MSAL/src/native_auth/network/responses/validator/token/MSALNativeAuthTokenResponseValidator.swift

```diff
@@ -139,6 +139,19 @@ final class MSALNativeAuthTokenResponseValidator: MSALNativeAuthTokenResponseVal
         apiError: MSALNativeAuthTokenResponseError,
         context: MSIDRequestContext
     ) -> MSALNativeAuthTokenValidatedResponse {
+        var apiError = apiError
+        if apiError.errorCodes?.contains(MSALNativeAuthESTSApiErrorCodes.resetPasswordRequired.rawValue) ?? false {
```

Comment:
This is more out curiosity but why is the MFA error for us handled in a different branch than in Android?